releases/worker.yaml

```diff
@@ -19,7 +19,7 @@ metadata:
 spec:
   releaseName: worker
   chart:
-    git: git@github.com:biomage-ltd/worker
+    git: git@github.com:hms-dbmi-cellenics/worker
```

Comment:
We are just wondering if this file is being used? There is a worker.yaml under both `releases/staging` and `releases/production` so we were unsure. Any ideas @ivababukova @marcellp @kafkasl ?